test/framework/service.go

```diff
@@ -37,6 +54,17 @@ func (f *Framework) doRequest(endpoint, method string, body []byte, query map[st
 	req := &http.Request{
 		URL:    u,
 		Method: method,
+		Header: make(http.Header),
+	}
+
+	if f.UseRouteForReportingAPI {
+		// check if the bearer token to the reporting-operator serviceaccount is uninitialized
+		if f.RouteBearerToken == "" {
+			return nil, 0, fmt.Errorf("f.UseRouteForReportingAPI is set to true, but f.RouteBearerToken is uninitialized: %v", err)
```

Comment:
Use the flag names instead of `f.UseRouteForReportingAPI` and `f.RouteBearerToken`